python/arcticdb/storage_fixtures/s3.py

```diff
@@ -12,27 +12,35 @@
 import os
 import re
 import sys
-import trustme
-import subprocess
 import platform
 from tempfile import mkdtemp
+import boto3
+import time
 
 
 import requests
-from typing import NamedTuple, Optional, Any, Type
+from typing import Optional, Any, Type
 
 from .api import *
 from .utils import get_ephemeral_port, GracefulProcessUtils, wait_for_server_to_come_up, safer_rmtree, get_ca_cert_for_testing
-from arcticc.pb2.storage_pb2 import EnvironmentConfigsMap
+from arcticc.pb2.storage_pb2 import EnvironmentConfigsMap, AWSAuthMethod
 from arcticdb.version_store.helper import add_s3_library_to_env
 
 # All storage client libraries to be imported on-demand to speed up start-up of ad-hoc test runs
 
-Key = NamedTuple("Key", [("id", str), ("secret", str), ("user_name", str)])
 _PermissionCapableFactory: Type["MotoS3StorageFixtureFactory"] = None  # To be set later
 
 logging.getLogger("botocore").setLevel(logging.INFO)
 
+S3_CONFIG_PATH = os.path.expanduser(os.path.join("~", ".aws", "config"))
+S3_BACKUP_CONFIG_PATH = os.path.expanduser(os.path.join("~", ".aws", "bk_config"))
```

Comment:
What's this for?

Reply:
A temporary config file for sts credential and profile used in the test